qemu/qemu.go

```diff
@@ -63,16 +64,16 @@ const (
 	NVDIMM DeviceDriver = "nvdimm"
 
 	// Virtio9P is the 9pfs device driver.
-	Virtio9P DeviceDriver = "virtio-9p-pci"
```

Comment:
This will break the ABI. I think you should consider adding a new one called Virtio9PGeneric